examples/mobile/UIComponents/components/gallery/gallery.js

```diff
@@ -60,8 +60,12 @@
 		var activeIndex = sectionChangerWidget.getActiveSectionIndex();
 
 		sectionsParentNode.removeChild(sections[activeIndex]);
-		sectionChangerWidget.refresh();
+
 		resetChildSectionInfo();
+		if (sectionsLength > 0) {
```

Comment:
Could we should update the
`sectionsLength = sections.length`

Reply:
The sectionsLength field is updated in resetChildSectionInfo function.

Reply:
ok.